other/buildTools/README.md

```diff
@@ -0,0 +1,38 @@
+# [SMF](www.simplemachines.org)
```

Comment:
We don't, this was primarily about getting it in and being able to work on it, it was just the one low-hanging fruit could definitely go without me even thinking about it too hard.